docker/docker-compose.yml

```
@@ -0,0 +1,32 @@
+version: '3'
+services:
+  snp_automate_2022:
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile
+      args:
+        - TAG=foxy-master
+    environment:
+      DISPLAY: $DISPLAY
+      XAUTHORITY: $XAUTHORITY
+      NVIDIA_DRIVER_CAPABILITIES: all
+      ROS_LOG_DIR: /tmp
+    container_name: snp_automate_2022
+    image: ghcr.io/ros-industrial-consortium/snp_automate_2022:foxy-master
+    stdin_open: true
+    tty: true
+    network_mode: host
+    privileged: true
+    user: ${CURRENT_UID}  # CURRENT_UID=$(id -u):$(id -g)
+    volumes:
+      - /tmp/.X11-unix:/tmp/.X11-unix
+      - /etc/hosts:/etc/hosts
+      - /etc/passwd:/etc/passwd:ro
+      - /etc/group:/etc/group:ro
```

Add `/dev` for USB devices

```suggestion
      - /etc/group:/etc/group:ro
      - /dev:/dev
```